OpenEdXMobile/res/layout/fragment_my_courses_list.xml

```diff
@@ -8,24 +8,29 @@
         android:layout_height="match_parent"
         android:background="@color/grey_act_background">
 
-        <androidx.swiperefreshlayout.widget.SwipeRefreshLayout
-            android:id="@+id/swipe_container"
+        <org.edx.mobile.view.custom.NestedScrollableHost
```

Comment:
IMO I would suggest using "ViewCompat.setNestedScrollingEnabled(ListObject, true);" instead of this in MyCourseListFragment.java at line 106
also, in CourseOutlineFragment.java at line 176

Reply:
Done.